package.json

```diff
@@ -105,6 +106,7 @@
     "jest-silent-reporter": "^0.6.0",
     "lint-staged": "^12.4.1",
     "lodash": "^4.17.21",
+    "luxon": "^3.5.0",
```

Comment:
Why was luxon added to the root package.json too? 🤔

Reply:
Unintended! Will revert.